In [3]:
import pandas

pandas.options.display.max_columns = None
pandas.options.display.max_rows = None

tvshow = pandas.read_csv("tv_shows.csv")
tvshow

Unnamed: 0    ID                                              Title  \
0              0     1                                       Breaking Bad   
1              1     2                                    Stranger Things   
2              2     3                                    Attack on Titan   
3              3     4                                   Better Call Saul   
4              4     5                                               Dark   
5              5     6                         Avatar: The Last Airbender   
6              6     7                                     Peaky Blinders   
7              7     8                                   The Walking Dead   
8              8     9                                       Black Mirror   
9              9    10                                 The Queen's Gambit   
10            10    11                                         Mindhunter   
11            11    12                                          Community   
12            12    13                                             Narcos   
13            13    14                                          Shameless   
14            14    15                                        Money Heist   
15            15    16                                 Marvel's Daredevil   
16            16    17                                            Lucifer   
17            17    18                                       Supernatural   
18            18    19                                        The Witcher   
19            19    20                                              Ozark   
20            20    21                                          The Crown   
21            21    22                                          Cobra Kai   
22            22    23                                      The Blacklist   
23            23    24                                          The Flash   
24            24    25                                          Outlander   
25            25    26                   Fullmetal Alchemist: Brotherhood   
26            26    27                                     The Good Place   
27            27    28                                     House of Cards   
28            28    29                                           Atypical   
29            29    30                                     Grey's Anatomy   
30            30    31                               Arrested Development   
31            31    32                                              Lupin   
32            32    33                               The Umbrella Academy   
33            33    34                            Orange Is the New Black   
34            34    35                                The Vampire Diaries   
35            35    36                     Demon Slayer: Kimetsu no Yaiba   
36            36    37                                     The Last Dance   
37            37    38                              American Horror Story   
38            38    39                                              Elite   
39            39    40                                    BoJack Horseman   
40            40    41                     Star Trek: The Next Generation   
41            41    42                                        Broadchurch   
42            42    43                                       The IT Crowd   
43            43    44                                   When They See Us   
44            44    45                               Love, Death & Robots   
45            45    46                                     Schitt's Creek   
46            46    47                    Marvel's Agents of S.H.I.E.L.D.   
47            47    48                                  Never Have I Ever   
48            48    49                                            The 100   
49            49    50                               American Crime Story   
50            50    51                                   The Last Kingdom   
51

In [45]:
netflix = tvshow[tvshow["Netflix"] == 1]
nshows = len(netflix)

hulu = tvshow[tvshow["Hulu"] == 1]
hshows = len(hulu)

primevideo = tvshow[tvshow["Prime Video"] == 1]
pshows = len(primevideo)

disney = tvshow[tvshow["Disney+"] == 1]
dshows = len(disney)

print("Netflix show count", nshows,"\nHulu show count", hshows, "\nPrime Video show count", pshows, "\nDisney+ show count", dshows)


Netflix show count 1971 
Hulu show count 1621 
Prime Video show count 1831 
Disney+ show count 351


# Finding which streaming platform had the most tv shows across Netflix, Hulu, Prime Video, and Disney+. My findings show that Netflix had the most tv shows total at 1971 with Prime Video at a close second with 1831 shows on their platform. That being said Disney+ had the least amount of shows at 351. 